v3/src/components/slider/slider-thumb.tsx

```diff
@@ -71,6 +69,8 @@ export const CodapSliderThumb = observer(function CodapSliderThumb({sliderContai
     setIsDragging(true)
   }
 
+  useSliderAnimation({sliderModel, running, setRunning})
```

Comment:
Move this to the `SliderComponent` since there is no interaction with the `SliderThumb`.

Reply:
`useSliderAnimation` has a lot of references to `sliderModel` which would require a lot of checking and accomodating for when `sliderModel` is `undefined`